package.json

```diff
@@ -45,7 +45,7 @@
     "prettier": "prettier --list-different '{.storybook,packages}/**/*.{ts,tsx}'",
     "prettier-clean": "prettier --write '{.storybook,packages}/**/*.{ts,tsx}'",
     "lint": "yarn pre-test && eslint --ext .tsx,.ts ./packages/**",
-    "type-check": "lerna run type-check",
+    "type-check": "lerna run type-check --ignore @twilio-paste/core",
```

Comment:
Does this change need to be reverted?

Reply:
No, it's a limitation with `tsc`. Because we're not using typescript to bundle the JavaScript, we're using it to only create the declarations. When typescript is set to only emit declarations you can't run `tsc` as a type-check tool.

In the long run we can't use it with newer versions of Typescript anyway, and building and creating the type declarations is a perfectly fine type check in itself, so we'll likely remove the type-check task altogether at some point.

Reply:
Realized I don't need this after all as the script is removed